internal/pkg/local/paths.go

```diff
@@ -25,11 +28,30 @@ func (m *Manager) NewPathsGenerator(rename bool) *PathsGenerator {
 	return &PathsGenerator{Manager: m, rename: rename, processed: make(map[string]bool)}
 }
 
-func (g *PathsGenerator) Update(objectState model.ObjectState) error {
-	return g.doUpdate(objectState, nil)
+func (g *PathsGenerator) Add(objectState model.ObjectState) *PathsGenerator {
+	if g.invoked {
+		panic(fmt.Errorf(`cannot add more paths, PathsGenerator have already been invoked`))
+	}
+	g.toUpdate = append(g.toUpdate, objectState)
+	return g
+}
+
+func (g *PathsGenerator) Invoke() error {
```

Comment:
`PathsGenerator` som tiez trocha prerobil, ... pridana metoda `Invoke`.
Vyhoda je, ze sa vykonaju vsetky operacia a az tak sa vratia vsetky chyby, ... nespadne to po prvej chybe -> jednoduchsie debugovanie.